### PR TITLE
Release v1.0.0 (Do not merge)

### PR DESCRIPTION
**Pull request by shannonwells**
# Goal
The goal of this PR is to create the v1.0.0 release.

# Discussion

# Checklist
- [ ] Chain spec updated
- [ ] Weights updated

### CODE REVIEW

**Review by aramikm — COMMENTED** [review]
Overall looks good, 
Do you need to modify https://github.com/LibertyDSNP/frequency/blob/main/js/api-augment/package.json#L3?

File: common/helpers/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 name = "common-helpers"
 publish = false
 repository = "https://github.com/LibertyDSNP/frequency/"
-version = "0.0.0"
+version = "1.0.0+polkadot0.9.30"
```
Comment (aramikm) [inline common/helpers/Cargo.toml:10]: Do we know if there might be some problems down the road if we have versioning with `+polkadot0.9.30` in terms of problem parsing in some tools or etc.

Comment (shannonwells) [reply common/helpers/Cargo.toml:10]: See #874 and its referenced Issues for the public version of that discussion

**Comment by shannonwells** [comment]
@aramikm the version is actually set by setting an environment variable in the CI step [Release / Build JS API Augment](https://github.com/LibertyDSNP/frequency/actions/runs/3850015253/jobs/6560165847):

```
RUST_TOOLCHAIN: nightly-2022-09-22
    NEW_RELEASE_VERSION: v1.0.0
    NEW_RELEASE_TAG: v1.0.0
    BIN_DIR: target/production
    RELEASE_BIN_FILENAME: frequency.amd[6](https://github.com/LibertyDSNP/frequency/actions/runs/3850015253/jobs/6560165847#step:8:6)4
    NPM_CONFIG_USERCONFIG: /home/runner/work/_temp/.npmrc
    NODE_AUTH_TOKEN: XXXXX-XXXXX-XXXXX-XXXXX
```

**Review by aramikm — APPROVED** [review]
💯

**Comment by wilwade** [comment]
All good! v1.0.0 is starting to be deployed to Rococo!

**Comment by shannonwells** [comment]
Screenshot 2023-01-06 at 10 17 11 AM

**Pull request closed without merging**